frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -238,6 +240,17 @@ function Pages(): JSX.Element {
             )}
             <PageButton icon={<IconTools />} identifier={Scene.ToolbarLaunch} to={urls.toolbarLaunch()} />
             <PageButton icon={<IconSettings />} identifier={Scene.ProjectSettings} to={urls.projectSettings()} />
+            {user?.is_staff && (
```

Comment:
Also I would consider adding a tooltip to the button, or page caption, explaining more-or-less "You have access to this because you're a staff user. [See PostHog docs on instance management.](https://www.youtube.com/watch?v=dQw4w9WgXcQ)"